src/components/SurveyComponent.vue

```diff
@@ -1,324 +1,277 @@
 <template>
-  <div id="surveyElement">
-    <survey v-if="ready" :survey="survey"></survey>
+  <div id="surveyElement" class="ma-4">
+    <survey v-if="ready" :survey="survey" :css="themes"></survey>
     <div v-if="ready" id="surveyResult"></div>
+    <v-progress-circular :value="100" v-if="!error && !ready" indeterminate
+      color="primary"></v-progress-circular>
+    <v-alert color="error" v-if="error" dark>
+      Error loading the application. See console for detail.
+      <div v-html="error"></div>
+    </v-alert>
   </div>
 </template>
 
 <script>
-
 import converter from 'questionnaire-to-survey';
 import { getScreeningInstrument } from '../util/screening-selector.js';
-import Worker from "../../node_modules/cql-worker/src/cql.worker.js"; // https://github.com/webpack-contrib/worker-loader
+import Worker from "../../node_modules/cql-worker/src/cql.worker.js"; //https://github.com/webpack-contrib/worker-loader
```

Comment:
Minor nitpick: removing a superfluous change to the original. I'm probably being overly paranoid, but might be conflicts/rebasing easier.
```suggestion
import Worker from "../../node_modules/cql-worker/src/cql.worker.js"; // https://github.com/webpack-contrib/worker-loader
```

Reply:
Also, try the **Commit suggestion** button if you haven't already!!